pkg/apis/options/providers.go

```diff
@@ -59,7 +59,9 @@ type Provider struct {
 	// CAFiles is a list of paths to CA certificates that should be used when connecting to the provider.
 	// If not specified, the default Go trust sources are used instead
 	CAFiles []string `json:"caFiles,omitempty"`
-
+	// CAFilesAppend determines if CAFiles are appended to the default store
+	// If set to true, CAFiles will be appended to the default Go trust stores, otherwise it will replace it
+	CAFilesAppend bool `json:"caFilesAppend,omitempty"`
```

Comment:
```suggestion
	// UseSystemTrustStore determines if your custom CA files and the system trust store are used
	// If set to true, your custom CA files and the system trust store are used otherwise only your custom CA files.
	UseSystemTrustStore bool `json:"useSystemTrustStore,omitempty"`
```